Aave Funding Updates (#33)
bgd-labs/aave-proposals-v3: src/20231102_Multi_AaveFundingUpdates/AaveFundingUpdates.md

## Motivation

This proposal intends to ensure the DAO is well capitalised. A separate DAO budget publication will be shared during Q4 2023.

The DAO has sufficient USDC, DAI and USDT deposited across the various instances of Aave Protocol to support existing streams and the AURA purchase 3.

However, with the emergence of new service provider proposal expected, the DAO needs to adjust its stable coin holdings. This publication supports transferring additional funds from Polygon to Mainnet and acquiring GHO on the market to ensure the DAO is well capitalised for the next 6 months. It also aims to move assets from the old V2 to the newer V3 aToken.

## Specification

- Deposit 1M units of DAI from Treasury into Aave v3
- Migrate all aDAI from Aave v2 to v3

- Redeem and transfer 1.7M USDC to Ethereum Treasury
- Redeem and transfer 0.5M DAI to Ethereum Treasury
- Redeem and transfer 0.75M USDT to Ethereum Treasury

> Swap the following asset holdings to GHO:

Reviewer: ARC USDC missed here

Author: hey @rustboyar is there a tool to do the ARC transfer and withdrawal?
the owner of the funds to Transfer is GovernanceV3, while to withdraw from ARC, it's the ARC timelock.
I think I have to do it in 2 steps, a payload to transfer, and then a payload to withdraw, so was hoping maybe there's a way to do both?

Author: removing ARC 'cause it's just a painful process

---

## Simple Summary

This AIP aims to consolidate Aave DAO's stablecoin holdings. It is divided into different parts that are detailed below. Part one is depositing DAI into aEthDAI, part two is migrating aDAI and aUSDT into aEthDAI and aEthUSDT, part three bridges assets from Polygon to Mainnet and part four swaps some holdings into GHO.

## Motivation

This proposal intends to ensure the DAO is well capitalised. A separate DAO budget publication will be shared during Q4 2023.

The DAO has sufficient USDC, DAI and USDT deposited across the various instances of Aave Protocol to support existing streams and the AURA purchase 3.

However, with the emergence of new service provider proposal expected, the DAO needs to adjust its stable coin holdings. This publication supports transferring additional funds from Polygon to Mainnet and acquiring GHO on the market to ensure the DAO is well capitalised for the next 6 months. It also aims to move assets from the old V2 to the newer V3 aToken.

## Specification

- Deposit 1M units of DAI from treasury (Aave V3 Ethereum Collector) into Aave v3 Ethereum
- Migrate all aDAI from Aave v2 Ethereum to Aave v3 Ethereum

> - Redeem from Aave v2 Polygon 1.7M USDC and bridge to Ethereum treasury

Reviewer 2: ```suggestion
- Redeem 1.7M USDC from Aave v2 Polygon and bridge to Ethereum treasury
```

> - Redeem from Aave v2 Polygon 0.5M DAI and bridge to Ethereum treasury

Reviewer 2: ```suggestion
- Redeem 0.5M DAI from Aave v2 Polygon and bridge to Ethereum treasury
```

> - Redeem from Aave v2 Polygon 0.75M USDT and bridge to Ethereum treasury

Reviewer 2: ```suggestion
- Redeem 0.75M USDT from Aave v2 Polygon and bridge to Ethereum treasury
```

Swap the following asset holdings to GHO on Ethereum mainnet:

> - All available aTUSD that can be redeemed (57,373.85 units at time of writing)

Reviewer 2: ```suggestion
- All available aTUSD that can be redeemed (~175,000 units at time of writing)
```

- 500k units of DAI held in treasury (received from previous Polygon bridge)

> - All available aBUSD (~42,500 units at time of writing)

Reviewer 2: ```suggestion
- All available aBUSD (~72,000 units at time of writing)
```

Swap the following asset holdings on Ethereum treasury to USDC (there is no GHO / ETH oracle thus these tokens cannot be swapped to GHO directly, they are Aave v2 Assets with no Aave v3 equivalent oracle):

> - All GUSD (18,464.03 units at time of writing)

Reviewer 2: ```suggestion
- All GUSD (18,464.03)
```
it's underlying so won't change

> - All UST (893,257.63 units at time of writing)

Reviewer 2: ```suggestion
- All UST (893,257.63)
```
